app/views/cameras/index.html.erb

```diff
@@ -2,7 +2,11 @@
   <h1>Cameras</h1>
   <ul>
     <% @cameras.each do |camera| %>
-      <li><%= camera.name %></li>
+      <li>
+        <%= camera.name %>
+        <%= link_to 'Edit', edit_camera_path(camera) %>
```

Comment:
- You have to make sure only the user that owns the camera, can edit or destroy the camera
- You can get the authenticated user, with: `current_user`, then you compare with `camera.user`